src/components/DetailList.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { getCovidFromApi, getCountry } from '../redux/actions/fetchApi';
+import Details from './Details';
+
+const DetailsList = () => {
+  const dispatch = useDispatch();
+  const dataCovid = useSelector((state) => state.covidReducer);
+  useEffect(() => {
+    if (dataCovid.country.length === 0) {
+      dispatch(getCovidFromApi());
+    }
+  }, []);
+  const todayDate = new Date().toISOString().slice(0, 10);
+  const handleCountry = () => {
+    const country = useParams();
+    const hello = country.country;
+    // const num = document.getElementById('date').value;
```

Comment:
Let's remove all commented code, to make the code cleaner and professional 🙂 

Kindly check for other occurrences 👍